### DISCUSSION TITLE
Kafka monitoring

### DISCUSSION THREAD

**Discussion by p4ssol4rgo** [edenhill/librdkafka#3518] [Q&A]
Hi all, 

I'm trying to figure out how to find these metrics https://docs.confluent.io/platform/current/kafka/monitoring.html in the statis json sent by the librdkakfa https://github.com/edenhill/librdkafka/blob/master/STATISTICS.md. For example "records-lead" or "records-lag-avg" in partition level. Or if I could get kafka monitoring meters by other way.

Thanks in advance.

**Comment by p4ssol4rgo** [comment]
@edenhill

**Reply by edenhill** [reply]
The first link is for metrics in the Java clients, not librdkafka.  But librdkafka does provide consumer_lag per partition for all currently assigned/consumed partitions.
